examples/Converting any jupyter notebook into .deepnote/README.md

```
@@ -0,0 +1,65 @@
+
+- `-o, --outputPath <path>`
+  Where to save the `.deepnote` file. If you provide a directory, the filename is auto-generated.
+
+## Open in Deepnote
+
+Once you have the `.deepnote` file, import it into Deepnote (e.g., via the "Import project" action in the Deepnote UI) or drag and drop the file into your workspace.
+
+## Troubleshooting
+
+- If `npx` is unavailable, install the CLI globally:
+  ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)